docs/release-notes/release-notes-0.9.1.md

```diff
@@ -0,0 +1,156 @@
+
+`8c3ae11` fix maker selection of ioauth input with expired timelocked addresses
+
+
+#### Documentation updates to clarify aspects of fidelity bond usage
+
+Questions and interactions with users post-the release of 0.9.0 made us realize that some details were not as clear as they should be. For example,
+the fact that only one fidelity bond will be advertised at a time:
+
+`f7180b8` Emphasize that ygen uses only single UTXO as fbond
+
+`148f970` Add single UTXO fb warning to gettimelockaddress
+
+Additionally, a couple of changes were needed to the doc to explain exactly how a user can/cannot spend fidelity bond utxos, after they've passed their timelock and are spendable on-chain (see above code commits for more detail).
+
+`e7c3c4e` detailed conditions for spending timelocked coins
+
+`13ae0c3` Remove doc saying timelock utxo can be spent in cj
+
+Giving users additional advice on the non-obvious question of how time-unlocked (previous fidelity bond) utxos should be spent:
+
+`cea80e2` Suggest use of coin control in fidelity-bond.md
+
+A typo fix in the equation defining fidelity bond value:
+
+`87f36c2` Fix error in simplified equation in fidelity-bond
+
+As per commit comment:
+
+`a090293` Emphasize the sybil protection gained by takers
+
+
+### Fix overflow error on 32 bit ARM
+
+See #953 for context. 32 bit systems hit a limit in datetime processing at year 2038 (see also next section). The fix here rewrites several datetime related calls to be compatible with those systems.
+
+`f9ea973` fix overflow error on 32bit ARM
+
+
+### Ob-watcher fixes
+
+As per above:
+`ce08a0c` Fix year 2038 problem in ob-watcher /fidelitybonds
+
+Also, a separate bug was fixed in the code triggered by the json export of the orderbook:
+
+`58d3e59` Fix Export Orders page on ob-watcher
+
+
+### More minor improvements and fixes
+
+Only in testing should a user be allowed to set a timelock in the past, for real usage it's an error:
+
+`79a330e` Check that gettimelockaddress argument is in future
+
+`53c7c7d` Better error message in case of invalid joinmarket.cfg format
+
+`c20655c` Always cast pid read from wallet lockfile to int
+
+`1bf9f85` Update link to irc logs to https
+
+This last item in the list is a bit more significant, as it was a potential crash vector in running takers, but it was rare and trivial to fix: a logging statement was crashing due to trying to use a negative satoshi value:
+
+`4586e34` Prevent crash if negative change in sweep
+
+
+### Testing and dev related background refactoring
+
+`c2312a4` clean up timestamp_to_time_number()
+
+`dc13038` reuse index for timenumber in FidelityBonds wallet
+
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release -
+
+- @openoms
+- @kristapsk
+- @chris-belcher
+- @PulpCattel
+- @undeath
+- @AdamISZ
+
+And thanks also to those who submitted bug reports, reviewed and otherwise helped out.
+
+
```

Comment:
nit - remove extra blank lines? :)

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/commit/b5c01c4c751977d10e847d761b99d5657b5fbc04